build/initialize-pipeline.ps1

```diff
@@ -4,11 +4,11 @@ $bypassPackaging = $true
 $suffix = "-ci"
 Write-Host "SourceBranch: $sourceBranch, Build reason: $buildReason"
 
-if($sourceBranch.endsWith('release/2.0') -or $sourceBranch.startsWith('release/2.0-hotfix')) {
+if($sourceBranch.Contains("release/2.0")) {
```

Comment:
This should probably be applied to any of the `release/` branches, as they exist for release builds only.

Reply:
done